### PR TITLE
Document known discoverable zwave_js devices

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Document known discoverable zwave_js devices

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54938
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -47,6 +47,18 @@ Note: A new network key is automatically generated for you. If this Z-Wave stick
 
 If you do not run Home Assistant OS (the default installation type) or Home Assistant Supervised, please see the [advanced installation instructions](#advanced-installation-instructions).
 
+### Discovery via USB
+
+Some devices can be auto-discovered, which can simplify the Z-Wave JS setup process. The following devices have been tested with discovery, and offer a quick setup experience:
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown:52]: Oxford comma, please void those.

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -47,6 +47,18 @@ Note: A new network key is automatically generated for you. If this Z-Wave stick
...
+
+| Device | Identifier | Vendor |
+| -------| ---------- | ------ |
+| Aeotec Z-Stick Gen5+ | 0658:0200 | https://aeotec.com/z-wave-usb-stick/ |
+| Nortek HUSBZB-1 | 10C4:8A2A | https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/ |
+| Zooz ZST10 | 10C4:EA60 | https://www.getzooz.com/zooz-zst10-s2-stick.html |
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown:58]: We should not use bare URLs, they render really badly. Instead, link the device names maybe?

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -47,6 +47,18 @@ Note: A new network key is automatically generated for you. If this Z-Wave stick
...
+
+Additional devices may be discoverable, however only devices that have been confirmed discoverable are listed above.
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown:60]: Missing comma after "however": `however, only`

**Pull request merged**